radicle-httpd/src/api.rs

```diff
@@ -49,10 +49,12 @@ impl Context {
         let repo = storage.repository(id)?;
         let (_, head) = repo.head()?;
         let payload = repo.project_of(self.profile.id())?;
```

Comment:
You can get this from the doc now, with `doc.project()`

Reply:
maybe also add a `let delegates = ...` in order to keep the format clean

Reply:
You have it already:
```
        let doc = repo.identity_of(self.profile.id())?;
```